src/containers/BooksList.js

```diff
@@ -0,0 +1,24 @@
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from '../components/Book';
+
+const BookList = (props) => {
+  const { books } = props;
+  return (
+    <table>
+      <th>Book ID</th>
+      <th>Title</th>
+      <th>Category</th>
+      { books.map((abook) => (<Book book={abook} key={abook} />))}
+
```

Comment:
Kindly remove the white empty space which will make your code looks more professional and readable